unittests/kernelcache.py

```diff
@@ -0,0 +1,191 @@
+import os
+import platform
+import pytest
+import requests
+import sys
+import tempfile
+import urllib3
+import warnings
+
+try:
+    cwd = os.environ['CSPYCE_TEST_KERNELS']
+except KeyError:
+    cwd = "/tmp" if platform.system() == "Darwin" else tempfile.gettempdir()
+    
+server = 'https://pds-rings.seti.org/testrunner_support/cspyce-unit-test-kernels/'
+
+warnings.simplefilter('ignore', urllib3.exceptions.InsecureRequestWarning)
+
+
+def get_kernel_name_from_url(url: str) -> str:
+    return url.split("/")[-1]
+
+
+def get_path_from_url(url: str) -> str:
+    return os.path.join(cwd, get_kernel_name_from_url(url))
+
+
+class CassiniKernels(object):
+    cassPck_url = "https://pds-rings.seti.org/testrunner_support/cspyce-unit-test-kernels/cpck05Mar2004.tpc"
```

Comment:
You defined `KERNEL_URL_ROOT` above, so you should use it...

```
cassPck_url = f"{KERNEL_URL_ROOT}cpck05Mar2004.tpc"
```

I find this a little hard to read. I would remove the `/` from the end of `KERNEL_URL_ROOT` and put the `/` here so you get:

```
cassPck_url = f"{KERNEL_URL_ROOT}/cpck05Mar2004.tpc"
```